### DISCUSSION TITLE
I need help...

### DISCUSSION THREAD

**Discussion by DevonMKatYT** [geode-sdk/sdk#614] [General]
is it possible for someone to teach me how to make a geode mod... like in a guide format... please

**Comment by Weebifying** [comment]
https://docs.geode-sdk.org/getting-started/prerequisites

**Reply by DevonMKatYT** [reply]
Thanks!

**Comment by retcinder** [comment]
Theres some guides at https://docs.geode-sdk.org/

**Reply by Weebifying** [reply]
🐢

**Reply by retcinder** [reply]
you're too fast at typing, weebifying

**Reply by DevonMKatYT** [reply]
Thanks!

**Reply by DevonMKatYT** [reply]
wait how do i open cmake

**Comment by SpaghettDev** [comment]
consider joining the [geode discord](https://discord.gg/9e43WMKzhp) and spam questions there as well 👍

**Reply by DevonMKatYT** [reply]
alr
